social-rs/server/src/user.rs

```diff
@@ -6,6 +6,7 @@ use uuid::Uuid;
 #[derive(Serialize, Deserialize, Debug, PartialEq, Clone, Schema)]
 #[collection(users)]
 pub struct User {
+    #[owned_by(User)]
```

Comment:
needed this for collection name

Reply:
oh interesting - this might make the graph tricky since we would a User->User relationship. might make sense to have an explicit annotation but this seems to work for now